src/components/searchBar/index.tsx

```diff
@@ -1,23 +1,86 @@
 import React, { useCallback, useState } from "react";
 
+interface SearchResultListProps {
+  id: number;
+  name: string;
+}
+
 export default function SearchBar() {
   const [value, setValue] = useState("");
+  const [searchResultList, setSearchResultList] = useState<SearchResultListProps[]>([]);
+  const [selectedIngredientsList, setSelectedIngredientsList] = useState<SearchResultListProps[]>([]);
```

Comment:
Naming suggestions:
```suggestion
  const [selectedSearchResults, setSelectedSearchResults] = useState<SearchResult[]>([]);
```
or
```suggestion
  const [selectedIngredients, setSelectedIngredients] = useState<Ingredient[]>([]);
```